datalad/distribution/tests/test_update.py

```diff
@@ -131,6 +131,11 @@ def test_update_fetch_all(src, remote_1, remote_2):
         f.write("different file load")
     rmt2.add("second.txt", git=True, commit=True, msg="Add file to git.")
 
+    # Let's init some special remote which we couldn't really update/fetch
+    if not os.environ.get('DATALAD_TESTS_DATALADREMOTE'):
```

Comment:
There is no additional harm in ignoring it again. But you are the one adding most of the environment switches. And you don't use the config mechanism hence the situation you observe will never change.